<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabRaspberryConnection</class>
 <widget class="QWidget" name="TabRaspberryConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Automatic Search For Available Raspberry on the Network</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btSearch">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btStopSearch">
          <property name="text">
           <string>Stop Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Devices Found - (Double click to stabilish connection to a Raspberry Pi)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="lvDevicesFound"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Device Chosen:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leDeviceChosen"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btDisconnect">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Logs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="teLogs"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
